src/components/NavigationButton/NavigationButton.js

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+NavigationButton.propTypes = {
+  onClick: PropTypes.func,
+  children: PropTypes.string,
+}
+
+export default function NavigationButton({ onClick, children }) {
+  return <StyledButton onClick={onClick}>{children.toUpperCase()}</StyledButton>
```

Comment:
This will crash when someone inserts an image into the button. Can we type children props with prop-types? 🤔

Reply:
Very good point.
I added `.isRequired`

Reply:
Have you tried this out, do we get a invalid prop types warning when we insert something other than a string (asking for a friend)